opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java

```diff
@@ -44,18 +44,18 @@ public void testAsyncError() {
      */
     @Test
     public void testAsyncErrorTestSpanReported() {
-        try {
-            // disable retry otherwise there can be 2 spans
-            CloseableHttpClient build = HttpClientBuilder.create().disableAutomaticRetries().build();
-            Client client = new ResteasyClientBuilder().httpEngine(new ApacheHttpClient4Engine(build)).build();
-            Response response = client.target(url("/asyncError"))
-                .request()
-                .get();
-            response.close();
+        // disable retry otherwise there can be 2 spans
+        CloseableHttpClient build = HttpClientBuilder.create().disableAutomaticRetries().build();
+        Client client = new ResteasyClientBuilder().httpEngine(new ApacheHttpClient4Engine(build)).build();
+        try (Response response = client.target(url("/asyncError"))
+            .request()
+            .get()) {
             response.readEntity(String.class);
             client.close();
```

Comment:
You shouldn't have the `client.close()` here, as it's in the finally block already

Reply:
oh my bad

Reply:
oh my bad